.github/workflows/deploy.yml

```
@@ -80,15 +84,15 @@ jobs:
           target: ${{ matrix.target }}
 
       - name: Setup | musl tools
-        if: matrix.target == 'x86_64-unknown-linux-musl'
+        if: matrix.target == 'x86_64-unknown-linux-musl' || matrix.target == 'i686-unknown-linux-musl'
```

Seems to build fine for me locally so this might not break CI.

For the future I think rewriting the condition to something like this is better (I hope this syntax is correct):
```suggestion
        if:  ${{ endsWith(matrix.target, 'musl') }}
```